settings.gradle.kts

```diff
@@ -82,4 +82,5 @@ fun File.projectsAndModules(): Pair<Set<String>, Set<String>> {
   val formattedProjects = projects.map { it.replace(":", "/") }.toSet()
   return Pair(formattedProjects, modules)
 }
+include(":ui:widgets")
 include(":ui:arch")
```

Comment:
this is not needed, since the logic in this file includes every module automatically